game/src/main/java/net/demilich/metastone/game/spells/trigger/AfterTurnStartTrigger.java

```diff
@@ -0,0 +1,36 @@
+package net.demilich.metastone.game.spells.trigger;
+
+import net.demilich.metastone.game.entities.Entity;
+import net.demilich.metastone.game.events.GameEvent;
+import net.demilich.metastone.game.events.GameEventType;
+import net.demilich.metastone.game.spells.TargetPlayer;
+import net.demilich.metastone.game.spells.desc.trigger.EventTriggerArg;
+import net.demilich.metastone.game.spells.desc.trigger.EventTriggerDesc;
+
+public class AfterTurnStartTrigger extends EventTrigger {
```

Comment:
Maybe needs to be retired given the card can be implemented maybe with existing stuff.